packages/Charts/src/demos/demo-data.ts

```diff
@@ -0,0 +1,263 @@
+import * as DG from 'datagrok-api/dg';
+
+export const energyUK = DG.DataFrame.fromCsv(
```

Comment:
Why do you use this way of storing demo tables? For data that isn't present in the demo files connection, there's an option to use [package files API](https://github.com/datagrok-ai/public/blob/master/help/develop/how-to/access-data.md#package-files). It seems that the tables are mostly from the demo connection, so you can simply open it via API, e.g.: `grok.data.getDemoTable('geo/earthquakes.csv')`

Reply:
The same goes for the test data. The package is big enough already, so unless there's a reason to add data this way, I don't see why we need these files here.

Reply:
thank you, guess you are right, didn't think about `grok.data.getDemoTable()`
will fix it